src/video/camerasource.cpp

```diff
@@ -214,14 +233,18 @@ void CameraSource::unsubscribe()
     }
     else
     {
+        //qDebug() << "second block";
         device->close();
         biglock = false;
     }
+    subscriptions--;
```

Comment:
> (compilers really aren't "smart" enough to optimize it out?)

 They are.

This issue is BS, code is for programmers, such nano-optimizations are up to compiler, if you want more of them in binary file use `-O3`. I'm used to i++, it's more readable for me, so I will keep using it.